### PR TITLE
Add linux question to create a new file and open in vi editor

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/linux-quiz.ts [hunk 1]
```diff
@@ -1169,6 +1169,17 @@ const linuxQuiz = [
...
+  {
+    Question:
+      "In Linux, which command is used to create a new file and open it in the Vim editor?",
+    Answer: "vi filename.extention",
+    Distractor1: "new filename.extention",
+    Distractor2: "create filename",
+    Distractor3: "None of the above",
+    Explanation:
+      "This command is used to create a new file using terminal and open it in the Vim editor.",
+    Link: ""
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1182]: The link field cannot be blank

```suggestion
    Link: "https://www.freecodecamp.org/news/learn-vim-beginners-tutorial/"
```

File: src/data/linux-quiz.ts [hunk 2]
```diff
@@ -1169,6 +1169,17 @@ const linuxQuiz = [
...
+  },
+  {
+    Question:
+      "In Linux, which command is used to create a new file and open it in the Vim editor?",
+    Answer: "vi filename.extention",
+    Distractor1: "new filename.extention",
+    Distractor2: "create filename",
+    Distractor3: "None of the above",
+    Explanation:
+      "This command is used to create a new file using terminal and open it in the Vim editor.",
```
Comment (jdwilkin4) [inline src/data/linux-quiz.ts:1181]:
```suggestion
      "vi filename.extention is used to create a new file using the terminal and open it in the Vim editor.",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Review by Shaik-mohd-huzaifa — COMMENTED** [review]
Applied Changes

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**